diff --git a/packages/core/routes/healthz.ts b/packages/core/routes/healthz.ts
index 1b971e6..a2e6c6e 100644
--- a/packages/core/routes/healthz.ts
+++ b/packages/core/routes/healthz.ts
@@ -1,5 +1,4 @@
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-const express = require('express');
+import express from 'express';
 const router = express.Router();
 
 router.get('/healthz', async (_req: any, res: any) => {
